src/store/models/modals.ts

```diff
@@ -81,6 +88,7 @@ export interface ModalsModel {
   assetInfo: AssetInfoModel;
   mintAsset: MintAssetModel;
   newAddress: NewAddressModel;
+  changeTaroBackend: ChangeTaroBackendModel;
```

Comment:
nit: to be more organized in this file, keep the order of the types/fields/actions consistent. Place all the new stuff at the botton of their groups.